armory/utils/export.py

```diff
@@ -131,6 +132,12 @@ def get_sample(x_i):
 
 
 class ObjectDetectionExporter(ImageClassificationExporter):
+    def __init__(self, base_output_dir, export_kwargs={}):
+        super().__init__(base_output_dir, export_kwargs)
+        self.ground_truth_coco_data = []
```

Comment:
I think the `coco` variable names are a bit confusing. I think `self.benign_pred_boxes_coco_format`, `self.ground_truth_boxes_coco_format`, self.adv_pred_boxes_coco_format` are more descriptive of their purpose